### PR TITLE
ZCS-2921 add imapd configuration options

### PR DESCRIPTION
**Pull request by tmclane**

### CODE REVIEW

**Comment by fatkudu** [comment]
Could you retarget this on develop please?

File: imap.adoc [hunk 1]
```diff
@@ -48,3 +48,12 @@ Do NOT configure a load balancer between the Zimbra HTTP Proxy node and any remo
 [IMPORTANT]
 The zmlocalconfig setting *nio_imap_enabled* MUST have value *true* when using the Zimbra IMAPD server.  The legacy
  Imap server which does not use NIO is not supported.
+
+==  IMAPD Configuration settings
+
+The following global and server level configuration attributes are available to control and tune the IMAPD service.
+
+* *zimbraRemoteImapServerEnabled*. When set to TRUE, the standalone IMAP server is enabled. When set to FALSE, the standalone IMAP server is disabled. Default value is FALSE.
```
Comment (plobbes) [inline imap.adoc]: nit: I prefer the use of em dash (--) over period (.) when separating the config key and the description (see https://github.com/Zimbra/adminguide/blob/master/backuprestore.adoc#backup-options for an example). It doesn't appear that we're always consistent with this however (perhaps something we could grep through the adoc's for and cleanup some time).

Also, the four (4) back to back *IMPORTANT* sections really look odd, but not sure how we'd like to fix that (or if we even want to... perhaps put that all in a table?)

File: imap_upstream.adoc [hunk 2]
```diff
@@ -16,13 +16,11 @@ IMAP related configuration options are available:
 
 The available balancing algorithms are as follows:
 
-* ClientIpHash
+* AccountIdHash
```
Comment (plobbes) [inline imap_upstream.adoc]: eww formatting...   This part, and just above should probably be redone with the use of bold (for zimbra config vars and perhaps italics (preferred? or underscore?) for AccountIdHash and custom and em dash (consistency please :)).

File: imap_upstream.adoc [hunk 3]
```diff
@@ -16,13 +16,11 @@ IMAP related configuration options are available:
...
 
-  Selects an IMAP server server based on a hash of the client IP address.
-  The hashing algorithm used is a simple modulus of the client's ip address as an integer with the number of
-   configured IMAP servers. This will always return the same 'index' for the same client ip address.
-
-   Note: Extra configuration is necessary if a NAT proxy is being utilized in the environment.
+  Selects an IMAP server server based on a hash of the account id of the user making the request.
+  The hashing algorithm used is a simple modulus of the account id as an integer with the number of
+   configured IMAP servers for the mailbox hosting the account. This will always return the same 'index' for the same account id.
```
Comment (plobbes) [inline imap_upstream.adoc]: nit: line lengths are a bit all over the place (not just here, but in our adocs in general), doesn't matter much, but I do find it somewhat annoying ... if you ever go look at something like linux kernel docs it seems like they try to at least shoot for the 80 char line length limit, I think something like that would be nice for our docs too.  Think emacs ```fill-paragraph``` or vim ```gq``` http://vim.wikia.com/wiki/Automatic_formatting_of_paragraphs

**Comment by tmclane** [comment]
Included in PR: https://github.com/Zimbra/adminguide/pull/49

**Pull request closed without merging**